validation/schema_testing/psc/pscStatementDeltaSchema_test.go

```diff
@@ -1,107 +1,135 @@
 package psc
 
 import (
-    "bytes"
-    "net/http/httptest"
-    "testing"
+	"bytes"
```

Comment:
Some of the changes to spacing here seem a little suspect to me. Could you have a look at them?